src/activities/content/ContentModuleEntity.js

```diff
@@ -94,7 +105,7 @@ export class ContentModuleEntity extends Entity {
 	equals(contentModule) {
 		const diffs = [
 			[this.title(), contentModule.title],
-			[this.descriptionRichText(), contentModule.descriptionRichText]
```

Comment:
do we want to get rid of `descriptionRichText` or have access to both?

Reply:
Since on load, we are now loading in the raw description into the editor, it will be "different" if we compare it with descriptionRichText. I.e loading a FACE page that contains replace strings then pressing cancel without making changes will prompt the discard changes if we keep that line

Reply:
ah i see i see, sorry you can ignore my comments i was misunderstanding what this section was doing

Reply:
that might make sense to keep things in sync. it does look a little confusing here when the diff is comparing different properties

Reply:
I'd say this is a nit. If you think about the scope of this entity, it does not know a "raw" or "editted" state for the content module description and it is just representing the data for consumers to use. In the LMS side of things it makes sense to keep the name raw because you are formatting the original description text, serializing it, and returning the response. Keeping `descriptionRichText` on the activity side (imo) makes more sense than raw as at the point it is rendering the description.